schema/cam/cam_schema_2-1-0.json

```diff
@@ -450,76 +450,79 @@
         },
         "low_frequency_container": {
           "type": "object",
-          "description": "The basic vehicle container low frequency",
-          "required": [
-            "exterior_lights",
-            "path_history"
+          "required":[
```

Comment:
Missing space before the opening square bracket.

Reply:
Ok, fixed